### ISSUE TITLE
Azure ARM template

### ISSUE THREAD

**Issue by karpinsn** [factoriotools/factorio-docker#408]
Over the weekend I wrote up an Azure ARM template to spin up an Azure storage account and container instance to host the docker container. I'm happy to shove this in my own repro but figured it might help others more in here. So my questions are:
1. Does the config file below here or in my own repo?
2. If the config file goes here any preference on where? I assume some sort of "configs" directory at the root of the repository?

**Comment by Fank** [comment]
I think yes why not, your idea is maybe related to https://github.com/factoriotools/factorio-docker/pull/227

**Comment by karpinsn** [comment]
Yep, saw that and figured it made sense just wasn't sure if that was the preferred route since that issue hadn't been touched in awhile. I'll get a PR going after work today.
